### ISSUE TITLE
UnicodeDecodeError when running unittests on Windows (cp1252)

### ISSUE THREAD

**Issue by crhowell** [jdepoix/youtube-transcript-api#83]
Attempting to run unittests or coverage with the following command from a **Windows** machine:
`coverage run -m unittest discover`

Results in :
```
cp1252.py", line 23, in decode
    return codecs.charmap_decode(input,self.errors,decoding_table)[0]
UnicodeDecodeError: 'charmap' codec can't decode byte 0x9d in position 119434: character maps to <undefined>
```

This is likely due to `test/test_api.py` attempting to open a file on [Lines 23-25](https://github.com/jdepoix/youtube-transcript-api/blob/master/youtube_transcript_api/test/test_api.py#L23-L25) without specifying an explicit encoding allowing the interpreter to just use whatever the System defaults to. 

Windows defaults to Windows-1252 character set.

I am logging the issue, though the fix is straight-forward and I will likely submit a fix for it soon as I work primarily from a Windows 10 machine.

**Comment by jdepoix** [comment]
Hi @crhowell, hope you're doing well! To be honest, I never ran this on windows or have really considered its implications. Thanks for reporting!

**Comment by jdepoix** [comment]
Fix merged in #84. Thanks to @crhowell!
